models/megengine_vision_frelu.md

```diff
@@ -0,0 +1,120 @@
+---
+template: hub1
+title: FReLU 
+summary:
+    en_US: "Funnel Activation for Visual Recognition"
+    zh_CN: FReLU （ImageNet 预训练权重）
+author: MegEngine Team
+tags: [vision, classification]
+github-link: https://github.com/megvii-model/FunnelAct
+---
+
+```python
+import megengine.hub
+model = megengine.hub.load('megvii-model/funnelact', 'resnet50_frelu', pretrained=True)
+model.eval()
+```
+<!-- section: zh_CN -->
+
+所有预训练模型希望数据被正确预处理。
+模型要求输入BGR的图片, 短边缩放到`256`, 并中心裁剪至`(224 x 224)`的大小，无需归一化处理。
+
+下面是一段处理一张图片的样例代码。
+
+```python
+# Download an example image from the megengine data website
+import urllib
+url, filename = ("https://data.megengine.org.cn/images/cat.jpg", "cat.jpg")
+try: urllib.URLopener().retrieve(url, filename)
+except: urllib.request.urlretrieve(url, filename)
+
+# Read and pre-process the image
+import cv2
+import numpy as np
+import megengine.data.transform as T
+import megengine.functional as F
+
+image = cv2.imread("cat.jpg").astype(np.float32)
+transform = T.Compose([
+    T.Resize(256),
+    T.CenterCrop(224),
+   # T.Normalize(mean=[103.530, 116.280, 123.675], std=[57.375, 57.120, 58.395]),  # BGR
+    T.ToMode("CHW"),
+])
+processed_img = transform.apply(image)[np.newaxis, :]  # CHW -> 1CHW
+logits = model(processed_img)
+probs = F.softmax(logits)
+print(probs)
+```
+
+### 模型描述
+
+目前我们提供了部分在ImageNet上的预训练模型(见下表)，各个网络结构在ImageNet验证集上的表现如下：
+
+|        Model             | Activation | Top-1 err.|
```

Comment:
@zhouyizhuang-megvii  should we use top-1 error or top-1 accuracy ?

Reply:
top-1 error is okay